plat/ti/k3/common/drivers/scmi/ti_scmi.mk

```diff
@@ -1,6 +1,7 @@
 PLAT_INCLUDES		+=	\
 				-Idrivers/scmi-msg/				\
 				-I${PLAT_PATH}/common/drivers/scmi		\
+				-I${PLAT_PATH}/board/am62l/scmi			\
```

Comment:
board specific files should not be part of this

Reply:
done